src/windows/main_window.py

```diff
@@ -465,6 +465,11 @@ def open_project(self, file_path, clear_thumbnails=True):
         app = get_app()
         _ = app._tr  # Get translation function
 
+        # First check for empty file_path (probably user cancellation)
+        if not file_path:
+            # Ignore the request
+            return
```

Comment:
I moved this check up here, because it felt cleaner than doing that `if file_path != ""` down below. (An empty string evaluates to false in Python, so this `if` is equivalent. Doing it sooner means we can bail on the whole thing, if there's no path to attempt loading anyway.)

Reply:
@ferdnyc - Definitely a good idea to bail on it sooner. 👍